AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -220,44 +215,36 @@ public async Task<IActionResult> CloneHearing(Guid hearingId, MultiHearingReques
             }
             catch (BookingsApiException e)
             {
-                _logger.LogError(e,
-                    "Failed to get hearing {Hearing}. Status Code {StatusCode} - Message {Message}",
+                _logger.LogError(e, "Failed to get hearing {Hearing}. Status Code {StatusCode} - Message {Message}",
                     hearingId, e.StatusCode, e.Response);
-                if (e.StatusCode != (int) HttpStatusCode.NotFound)
-                    throw;
-
+                if (e.StatusCode != (int) HttpStatusCode.NotFound) throw;
                 return NotFound($"No hearing with id found [{hearingId}]");
             }
 
             try
             {
                 var newParticipantList = new List<ParticipantRequest>();
-
-                if (CheckHearingWithinThirtyMinutes(originalHearing))
+                if (IsHearingStartingStartSoon(originalHearing) && originalHearing.Status == BookingStatus.Created &&
+                    !_hearingsService.IsAddingParticipantOnly(request, originalHearing))
+                {
+                    _logger.LogWarning("You can't edit a hearing within 30 minutes of it starting");
+                    ModelState.AddModelError(nameof(hearingId),
+                        $"You can't edit a hearing [{hearingId}] within 30 minutes of it starting");
+                    return BadRequest(ModelState);
+                }
+                else if(!IsHearingStartingStartSoon(originalHearing) || originalHearing.Status == BookingStatus.Booked)
```

Comment:
I'm not entirely clear on what BookingStatus.Created and BookingStatus.Booked signify. Does this distinguish between confirmed and unconfirmed bookings?

I know VIH-7556 used ConfirmedDate for this.

Reply:
yes, I found from ui side, if the booking status is created then set a confirmed flag there as the screen shot, please correct me if I'm wrong
![image](https://user-images.githubusercontent.com/47365487/118155362-133a5b00-b410-11eb-8c93-7055f2910cdc.png)

Reply:
Ok, that should fine then for this. In the future we should probably look to rename these to be more clear what they signify.